xwiki-rendering-syntaxes/xwiki-rendering-syntax-xwiki20/src/main/java/org/xwiki/rendering/internal/renderer/xwiki20/XWikiSyntaxChainingRenderer.java

```diff
@@ -229,7 +229,6 @@ public void beginFormat(Format format, Map<String, String> parameters)
         // If the previous format had parameters and the parameters are different from the current ones then close them
         if (this.previousFormatParameters != null) {
             if (parameters.isEmpty()) {
-                // print("(%%)");
                 // this.previousFormatParameters = null;
```

Comment:
Why remove the first comment and not the second one?

Reply:
Wait

Reply:
This is weird, I don't remember this modification when I merged the pull request.

Reply:
I put it back.

Reply:
It was a Sonar issue.

Reply:
http://sonar.xwiki.org/issues/search#issues=9a20df47-672f-4fca-9a38-767978c7d7e6

Reply:
It's not the right fix for it.